.github/workflows/qa.yaml

```diff
@@ -12,3 +12,11 @@ jobs:
           REQUIRE_DEV: true
         with:
           args: analyse -c .phpstan/phpstan.neon.dist
+  psalm:
+    name: Psalm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Psalm
+        uses: docker://vimeo/psalm-github-actions
```

Comment:
why do we need to execute it like this, if we have it on composer json, we could simply run it directly

Reply:
Indeed,
If we stop tu use psalm and phpstan github actions we could be sure that a developper can run static analysis locally.

Reply:
Imo we should do it for phpstan too (separare pr). I had a lot of trouble getting an env similar to what we do with github actions because of this.

Reply:
Can you please describe the problem? I don't understand what should I do wirh this PR.

Reply:
@jordisala1991 Is asking if we could use a php/composer actions where we run
- Composer install
- Psalm

Instead of the psalm actions.

This way we check that the local psalm is working. (and we require every needed plugin)

Reply:
not in my plans to do this in this PR. Can it be in separate PR

Reply:
I mean this is (this PR) is better then nothing.

Reply:
Sure, I won't see this as a blocking point. Maybe we should create an issue instead @jordisala1991 ?